Formula/gw.rb

```diff
@@ -0,0 +1,23 @@
+class Gw < Formula
+  desc "Browser for genomic sequencing data and variants"
+  homepage "https://github.com/kcleal/gw"
+  url "https://github.com/kcleal/gw/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "b479e19c2fed0704a652c0fcbd653517ce01ac429ec7753c95a583c35a733012"
+  license "MIT"
+  depends_on "fontconfig"
+  depends_on "glfw"
+  depends_on "htslib"
+  depends_on "icu4c"
+  depends_on "wget"
```

Comment:
Why does it need this

Reply:
Thanks for the help. I think all these are needed at some point during make. Possibly icu4 can be dropped. Is this the right way to add dependencies?

Reply:
What is it downloading in the install?

Reply:
It's downloading a pre-build binary - skia library

Reply:
I realised the GW repo only has 70stars on github, so it's not going to pass the build anyway?

Reply:
Prebuilt binaries aren't acceptable either